<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2508</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="open">
        <property name="text">
         <string>open all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>close all</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,0">
      <item row="0" column="2">
       <widget class="QComboBox" name="rightId"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="saveRight">
        <property name="text">
         <string>save right</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLCDNumber" name="rightLcdNumber"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="openRight">
        <property name="text">
         <string>open right</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="right">
        <property name="text">
         <string>right  id:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QPushButton" name="closeRight">
        <property name="text">
         <string>close right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="rightView">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QPushButton" name="saveLeft">
        <property name="text">
         <string>save left</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLCDNumber" name="leftLcdNumber"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QPushButton" name="closeLeft">
        <property name="text">
         <string>close left</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="left">
        <property name="text">
         <string>left  id:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="leftId"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="openLeft">
        <property name="text">
         <string>open left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="leftView">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="check">
        <property name="text">
         <string>find corner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save">
        <property name="text">
         <string>save all</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QPushButton" name="saveMiddle">
        <property name="text">
         <string>save middle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="openMiddle">
        <property name="text">
         <string>open middle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="closeMiddle">
        <property name="text">
         <string>close middle</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLCDNumber" name="middleLcdNumber"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="middleId"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="middleView">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="middle">
        <property name="text">
         <string>middle id:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="left2middle">
        <property name="text">
         <string>left-middle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="right2middle">
        <property name="text">
         <string>right-middle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2508</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="settings"/>
   </widget>
   <widget class="QMenu" name="tool">
    <property name="title">
     <string>工具</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>保存</string>
     </property>
     <addaction name="action_Image"/>
     <addaction name="action_Cloud"/>
     <addaction name="action_Depth"/>
    </widget>
    <addaction name="clearFile"/>
    <addaction name="menu"/>
    <addaction name="mergeCloud"/>
   </widget>
   <addaction name="file"/>
   <addaction name="tool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settings">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/img/settings.png</normaloff>:/icon/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="calibrateStereo">
   <property name="text">
    <string>双目标定</string>
   </property>
  </action>
  <action name="clearFile">
   <property name="text">
    <string>清空文件</string>
   </property>
  </action>
  <action name="action_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </action>
  <action name="action_Cloud">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>点云</string>
   </property>
  </action>
  <action name="action_Depth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>深度图</string>
   </property>
  </action>
  <action name="mergeCloud">
   <property name="text">
    <string>合并点云</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
